packages/runtime/src/plugins/utils/filterForSubtreeRoots.ts

```diff
@@ -0,0 +1,11 @@
+import type { Node, NodeEntry } from 'slate'
```

Comment:
These utils aren't all used here, but they are going to be used in the wip commit. They were c/p'd from the `ListPlugin/utils` folder